enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,346 @@
+
+_**Cons**_
+
+Etcd pod is a static-pod, and the availability of the etcd cluster is vital for cluster stability. As the Etcd cluster is being managed by the `Cluster-etcd-operator`, and a `Deployment` resource manages the _mirrored_ etcd pods, any changes in the pod manifest results into rolling out a new revision, which leads to a duration of `etcd-cluster` unavailability, till the new revision is rolled out for all etcd pods. Consequently, the `API-Server` can not process any write requests, and read requests are being handled; if any, from the API-Server cache.   
+
+The backup sidecar container is being `disabled` by default. Upon creating a `Backup` CR on an Openshift cluster, the `PeriodicBackupController` reacts by enabling the container. This leads to a change in etcd static pod manifest, and a new revision is being rolled out. As explained above, such action causes etcd-cluster and API-Server unavailability.
+
+Additionally, any error within the backup sidecar, the container goes into `CrashLoopBackOff`, which leads to the whole `Pod` reports `NotReady` status. Consequently, the whole `Etcd-cluster` can not server requests, nor the `API-Server`.
+
+
+#### DaemonSet manages an Etcd-backup pod within every control-plane node.
+
+This approach separates the `etcd-backup-server` deployment from the etcd static pods, to avoid affecting etcd-cluster and API-Server availability upon any error within the etcd-backup-server.
+
+The advantage of using `DaemonSet` over Deployment is to guarantee scheduling etcd-backup-server pods on all `Control-plane` nodes, regardless of the number of nodes (e.g. 1, 3, 5) using `nodeSelector` capability.
+By setting the `nodeSelector` to the master node label (i.e. `node-role.kubernetes.io/master`), etcd-backup-server's pods are guaranteed to scheduled on every control-plane node.
+
+This method has been implemented and tested. The same container used in the Sidecar approach has been utilised. However, the scheduling is being managed by a DaemonSet.
+Upon creating a `Backup` CR on an Openshift cluster, the `PeriodicBackupController` reacts by creating a DaemonSet resource, with the `etcd-backup-server` container and the configurations applied in the CR.
+During [testing](https://github.com/openshift/cluster-etcd-operator/pull/1354#issuecomment-2408661261), this method has shown a solid and stable deployment, where the Backup pods shown only `1` restart over a duration of `45` minutes. Moreover, the cause of the restart on all the Backup pods were due to `node` issue, not the backup container.
+
+_**Pros**_
+
+As the Backup pods are being scheduled independently of the etcd pods, any error within the Backup pods does not influence cluster Openshift cluster availability.
+
+Moreover, as the Backup pods are being scheduled within every master nodes, the backups are being taken from every local etcd member on the same node. This avoids overwhelming a specific member with backup requests.
```

Comment:
question: it has been a while but doesn't a snapshot request always go to the leader?